src/clj/qbits/jet/client/http.clj

```diff
@@ -356,18 +356,21 @@
   "Asynchronously loops and looks for presence of trailers in the response.
    When available, they are propagated to the trailers channel and other request channels closed.
    Please see https://github.com/eclipse/jetty.project/issues/3842 for details on why we need this."
-  [response-trailers-ch response-body-ch close-request-channels! response]
+  [response-trailers-ch response-body-ch content-provider-promise response]
   (async/go
-    (loop []
-      ;; iterate every second and check if trailers are available
-      (async/<! (async/timeout 1000))
-      (let [trailers (.getTrailers ^HttpResponse response)]
-        (if trailers
-          (do
-            (async/>! response-trailers-ch (util/http-fields->map trailers))
-            (close-request-channels!))
-          (when-not (protocols/closed? response-body-ch)
-            (recur)))))))
+    ;; empirically determine the timeout
```

Comment:
Can we explain what it means to empirically determine the timeout?

Reply:
Removed the comment (it was meant to be a `TODO shams` but I didn't label and forgot to remove). I had tried another approach which was still racy and had two timeouts with two loops. The comment was meant for that approach.